airflow/providers/amazon/aws/hooks/emr.py

```diff
@@ -254,11 +253,6 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         kwargs["client_type"] = "emr-serverless"
         super().__init__(*args, **kwargs)
 
-    @cached_property
-    def conn(self):
-        """Get the underlying boto3 EmrServerlessAPIService client (cached)"""
-        return super().conn
-
```

Comment:
nothing to do with the sauce, but I took the opportunity to clean this as I was around